packages/fluentui/docs/src/examples/components/Datepicker/States/DatepickerExampleClearable.shorthand.tsx

```diff
@@ -0,0 +1,6 @@
+import { Datepicker } from '@fluentui/react-northstar';
+import * as React from 'react';
+
+const DatepickerExampleClearable = () => <Datepicker clearable={true} today={new Date(2020, 9, 1, 0, 0, 0, 0)} />;
```

Comment:
```suggestion
const DatepickerExampleClearable = () => <Datepicker input={{clearable: true}} today={new Date(2020, 9, 1, 0, 0, 0, 0)} />;
```
I was thinking more of just passing `clearable` to the input slot, so we don't need to forward the props

Reply:
Makes sense. 

For `required` and `disabled`, I think we keep it explicit on the DatepickerProps, right?

Reply:
Yeah, those are really common usage, so I would keep them as is :)